cbrain_task/recon_all/bourreau/recon_all.rb

```diff
@@ -139,7 +147,23 @@ def cluster_commands #:nodoc:
 
     recon_all_command = "recon-all#{lbl_ext} #{with_qcache} #{with_mprage} #{with_3T_data} #{with_cw256} #{with_hippocampal} -sd . #{subjid_info} #{step} #{with_notal_check} #{lbl_options}"
 
+    # Copy license
+    cp_license = <<-CP_LICENSE
+
+# Handle FreeSurfer license
+
+if [ ! -f "$FREESURFER_HOME/license.txt" ] && [ ! -f "$FREESURFER_HOME/.license" ] ; then
+  echo Could not find a license installed.
+  if test -f license.txt ; then
+    echo Attempting to install the license
+    cp license.txt "$FREESURFER_HOME"
```

Comment:
Maybe exit immediately if the cp command fails?

`cp license.txt "$FREESURFER_HOME" || exit 20`